cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java

```diff
@@ -0,0 +1,90 @@
+package no.unit.nva.cristin.organization;
+
+
+import com.amazonaws.services.lambda.runtime.Context;
+import no.unit.nva.cristin.model.Organization;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.paths.UriWrapper;
+
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+import static no.unit.nva.cristin.projects.Constants.BASE_PATH;
+import static no.unit.nva.cristin.projects.Constants.DOMAIN_NAME;
+import static no.unit.nva.cristin.projects.Constants.HTTPS;
+import static no.unit.nva.cristin.projects.Constants.IDENTIFIER;
+import static no.unit.nva.cristin.projects.Constants.ORGANIZATION_PATH;
+import static no.unit.nva.cristin.projects.ErrorMessages.ERROR_MESSAGE_INVALID_PATH_PARAMETER_FOR_ID;
+import static no.unit.nva.cristin.projects.ErrorMessages.ERROR_MESSAGE_INVALID_QUERY_PARAMS_ON_LOOKUP;
+import static no.unit.nva.cristin.projects.ErrorMessages.ERROR_MESSAGE_LANGUAGE_INVALID;
```

Comment:
import from projects should not be in this module, maybe move them to commons

Reply:
Im surprised it works...

Reply:
Maybe put common error messages in commons module? If we use them in more than one module